docs/usage/exporting_models.md

```diff
@@ -290,7 +336,7 @@ print(m.model_dump_json())
 
 ## `pickle.dumps(model)`
 
-Using the same plumbing as `copy()`, *pydantic* models support efficient pickling and unpickling.
+Using the same plumbing as `model_copy()`, *pydantic* models support efficient pickling and unpickling.
 
 ```py test="skip"
```

Comment:
Did you see this? Just making sure.

Reply:
Yes, actually if you copy the code into a file it should work but it doesn't work in `test_docs_examples`
`_pickle.PicklingError: Can't pickle <class '__main__.FooBarModel'>: attribute lookup FooBarModel on __main__ failed`